tasks/request_status_for_job/request_status_for_job.py

```diff
@@ -29,12 +28,12 @@ def task(async_operation_id: str, db_connect_info: Dict) -> Dict[str, Any]:
                 granule_id (str)
                 status (str): pending|success|failed
     """
-    if async_operation_id is None or len(async_operation_id) == 0:
+    if job_id is None or len(job_id) == 0:
         raise ValueError(f"async_operation_id must be set to a non-empty value.")
```

Comment:
Possibly change out `async_operation_id` for `job_id` or `asyncOperationId` for the log.

Reply:
Whoops. Thanks.